packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -247,6 +235,12 @@ export const preparePortfolioKit = (
           valueShape: M.remotable('Position'),
         }),
         targetAllocation: undefined,
+        pendingEvmAccounts: harden({
+          Avalanche: { nonce: 1, lca: null, remoteAddress: null },
+          Arbitrum: { nonce: 1, lca: null, remoteAddress: null },
+          Polygon: { nonce: 1, lca: null, remoteAddress: null },
+          Optimism: { nonce: 1, lca: null, remoteAddress: null },
+        }),
```

Comment:
Maybe we can do something like:
```suggestion
        pendingEvmAccounts: harden(Object.fromEntries(
    keys(AxelarChain).map(chain => [
      chain,
      { nonce: 1n, status: 'pending', lca: null, remoteAddress: null }
    ])
  ))
```